README.md

```diff
@@ -48,9 +57,30 @@ You can find ways to improve the libraries in the [issues](https://github.com/Gl
 Direct pushing to the GlacioHack repository is not permitted.
 
 
-### Documentation - please read ! ###
-In the interest of keeping the documentation simple, please write all docstring in reStructuredText (https://docutils.sourceforge.io/rst.html) format - eventually, we will try to set up auto-documentation using sphinx and readthedocs, and this will help in that task.
+### Development installation
+* Clone the main repo (or your fork)
+```bash
+git clone https://github.com/GlacioHack/GeoUtils  # Or your own fork
```

Comment:
They should definitely fork the repository if they want to make any change as they won't be able to push to the main repo. 
Should we have something more detailed, similar to what I wrote for the GlacioHack? https://github.com/GlacioHack/xdem/wiki/Taking-part-to-a-GlacioHack

Reply:
Although that page describes how to contribute to both xdem and geoutils at the same time, so we should have a simpler version for geoutils.

Reply:
I mean they can clone the upstream and add their fork later, that's what I meant! Just for testing, they don't need a fork. It's when they want to contribute that this is needed. 

Maybe simple-turned-overly-complicated from my part? Shall I just say "clone your fork"?

Reply:
We should just update the contributing.md file accordingly I guess.

Reply:
And reference to it in the main README.

Reply:
Yeah makes sense. The install instructions for contributing should be in CONTRIBUTING.md, not the README haha

Reply:
There is already a dev-install instruction in CONTRIBUTING.md, so I just referenced to that file in 15fa5df